mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
so in sklearn and pandas people tend to return self,
in core Python None.

One problem I know realize is that we cannot really use .copy() elegantly with a None API ...